lib/tenstorrent/bh_arc/init.c

```diff
@@ -421,13 +422,14 @@ static int InitHW(void)
 
 	/* Load FW config, Read Only and Flash Info tables from SPI filesystem */
 	/* TODO: Add some kind of error handling if the load fails */
-	if (!IS_ENABLED(CONFIG_TT_SMC_RECOVERY)) {
-		load_fw_table(large_sram_buffer, SCRATCHPAD_SIZE);
-	}
-	load_read_only_table(large_sram_buffer, SCRATCHPAD_SIZE);
-	if (!IS_ENABLED(CONFIG_TT_SMC_RECOVERY)) {
-		load_flash_info_table(large_sram_buffer, SCRATCHPAD_SIZE);
-	}
+	/* if (!IS_ENABLED(CONFIG_TT_SMC_RECOVERY)) {
+	 * load_fw_table(large_sram_buffer, SCRATCHPAD_SIZE);
+	 * }
+	 * load_read_only_table(large_sram_buffer, SCRATCHPAD_SIZE);
+	 * if (!IS_ENABLED(CONFIG_TT_SMC_RECOVERY)) {
+	 * load_flash_info_table(large_sram_buffer, SCRATCHPAD_SIZE);
+	 * }
+	 */
```

Comment:
I think we can remove this - except we should preserve the behaviour of loading only the read-only table when `CONFIG_TT_SMC_RECOVERY` is set in `bh_fwtable.c::init()`